ops_manager.tf

```diff
@@ -47,6 +47,29 @@ resource "google_compute_instance" "ops-manager" {
   }
 }
 
+resource "google_compute_instance" "optional-ops-manager" {
+  name         = "${var.env_name}-optional-ops-manager"
+  depends_on   = ["google_compute_subnetwork.ops-manager-subnet"]
+  machine_type = "n1-standard-2"
+  zone         = "${element(var.zones, 1)}"
+  count        = "${var.optional_opsman_image_count}"
```

Comment:
You can remove this variable (reducing the complexity for users) and set `count` to 0 or 1 based on the absence or presence of `optional_opsman_image_name`:

`count = "${min(length(split("", var.optional_opsman_image_name)),1)}"`